pkg/storage/utils/decomposedfs/spaces.go

```diff
@@ -803,24 +803,32 @@ func (fs *Decomposedfs) storageSpaceFromNode(ctx context.Context, n *node.Node,
 		return nil, err
 	}
 
-	m := make(map[string]*provider.ResourcePermissions, len(grants))
+	pm := make(map[string]*provider.ResourcePermissions, len(grants))
```

Comment:
Sorry for the nitpicking, but I am not happy about these var names. Why not permissionsMap?